### PR TITLE
Added possibility to customize span operationName

### PR DESCRIPTION
**Pull request by m1x0n**
Extended `TracerConfig` with `OperationNameFunc` property which allows to override default operation name during span creation.

If there is no `OperationNameFunc` provided then fallback will be used which covers most of the use-cases.

### CODE REVIEW

File: jaegertracing/jaegertracing.go [hunk 1]
```diff
@@ -62,6 +62,9 @@ type (
 		// http body limit size (in bytes)
 		// NOTE: don't specify values larger than 60000 as jaeger can't handle values in span.LogKV larger than 60000 bytes
 		LimitSize int
+
+		// OperationNameFunc composes operation name based on context. Can be used to override default naming
+		OperationNameFunc func(ctx echo.Context) string
```
Comment (aldas) [inline jaegertracing/jaegertracing.go:67]: Please rename that parameter from `ctx` to `c`. Named parameters are handy when your IDE (for example GoLand by Jetbrains) does autocompletion and helps to autofill names. By convention `echo.Context` is usually named as `c`.

**Review by aldas — CHANGES_REQUESTED** [review]
Other than this one remark everything else seems ok!

File: jaegertracing/jaegertracing.go (hunk 1 above)
Comment (m1x0n) [reply jaegertracing/jaegertracing.go]: Done

**Review by aldas — APPROVED** [review]
lgtm

**Pull request merged**